app/controllers/applications_controller.rb

```diff
@@ -1,11 +1,11 @@
 class ApplicationsController < ApplicationController
   def show
     @application = Application.find(params[:id])
-    if params[:search].present?
-      @pets = Pet.search(params[:search])
-    else
-      @pets = Pet.adoptable
-    end
+    @pets = if params[:search].present?
+              Pet.search(params[:search])
+            else
+              []
```

Comment:
This looks a little fishy

Reply:
This is refactored by rubocop but it works for the purpose of the user story.

Reply:
Specifically, why has line 7 changed?

Reply:
changed to an open array so it could be filled as you add dogs to application